docs/libpod.conf.5.md

```diff
@@ -99,7 +99,7 @@ libpod to manage containers.
   a slirp4netns network.  If "" is used then the binary is looked up using the $PATH environment variable.
 
 **events_logger**=""
-  Default method to use when logging events. Valid values are "journald" and "file".
+  Default method to use when logging events. Valid values are "file", "journald", and "null".
```

Comment:
What does "null" mean?  If it means "none" should we use that?  Seems like a more Human name for it.

Reply:
After reading, I think we should present to the user NONE, Null is a programmer term and does not carry the same meaning as None.

Reply:
"null" as in "forward to /dev/null" - I'm open to changing this one, though. None also works.